src/Components/Filters/Filters.css

```diff
@@ -0,0 +1,32 @@
+.Filters {
+  flex-basis: 40%;
+}
+
+.filter-block {
```

Comment:
Request!
Please, use [BEM](http://getbem.com/introduction/) methodology for class naming, for ex.:
```
.filters {}
.filters__block {}
...
.block {}
```